Library/Homebrew/help.rb

```diff
@@ -39,6 +47,13 @@ def self.help(cmd = nil, empty_argv: false, usage_error: nil, remaining_args: []
       exit 0
     end
 
+    sig {
+      params(
+        cmd: String,
+        path: Pathname,
+        remaining_args: T.nilable(T::Array[String]),
```

Comment:
```suggestion
        remaining_args: T::Array[String],
```
feels nicer if possible given there's no default parameter